### PR TITLE
feat: Add tags and filtering feature

### PR DESCRIPTION
**Pull request by sasjo**

### CODE REVIEW

File: src/assets/css/main.css [hunk 1]
```diff
@@ -189,3 +189,41 @@ li {
 p {
   margin-bottom: 1rem;
 }
+
+ul.tags {
+  list-style: none;
+}
+
+ul.tags li {
```
Comment (tozz) [inline src/assets/css/main.css]: Invisible styling? ;)

**Comment by sasjo** [comment]
PR #52 should be merged into this one before merged to master.

File: src/js/components/Radar.jsx [hunk 2]
```diff
@@ -13,12 +14,22 @@ export default class Radar extends Component {
     }).isRequired,
   };
 
-  renderRadar = () => {
+  constructor(props) {
```
Comment (tozz) [inline src/js/components/Radar.jsx]: This constructor seems unnecessary, unless you really need to create the radar before componentDidMount.

File: src/js/modules/__mocks__/radarService.js [hunk 3]
```diff
@@ -25,6 +25,10 @@ const mockEntries = {
     ],
     quadrant: { index: 2, dirname: 'dev' },
     filename: 'java.html',
+    tags: [
+      '90\'s',
```
Comment (tozz) [inline src/js/modules/__mocks__/radarService.js]: 90's? :D

**Pull request merged**